www/_template/guides/svelte.md

```diff
@@ -1,12 +1,292 @@
+
+Image: GIF showing code and site side by side, site is a "Learn Svelte" link on a white background. When the text is edit to add "Hello world" and the file saves, the changes show up in the site immediately.
+
+## Customize your project layout
+
+Since you'll be adding a bunch of new files, you probably don't want them crowding up your top-level root directly. Snowpack is flexible enough to support whatever project layout that you prefer. In this guide, you'll learn how to use a popular project pattern from the Svelte community.
+
+```
+📁 src : your Svelte components and their assets (CSS, images)
+    ↳ index.js
+    ↳ App.svelte
+📁 public : global assets like images, fonts, icons, and global CSS
+    ↳ index.css
+    ↳ index.html
+```
+
+Use your favorite visual editor to rearrange and rename, or run these commands in the terminal:
+
+```bash
+mkdir src
+mkdir public
+mv index.js src/index.js
+mv App.svelte src/App.svelte
+mv index.html public/index.html
+mv index.css public/index.css
+```
+
+This means if you are running Snowpack right now, the site is now broken as the files are all in different places. Lets add a "mount" configuration to update your site to your new project layout.
+
+The `mount` configuration changes where Snowpack scan for and builds files. Head back to the `snowpack.config.js` file you edited when you added `@snowpack/plugin-svelte`. Add this to the empty `mount` object:
+
+```diff
+  mount: {
+-   /* ... */
++   // directory name: 'build directory'
++   public: '/',
++   src: '/_dist_',
+  },
+```
+
+<img src="/img/guides/react/folderstructure.png" alt="The original file configuration had Snowpack building the directory structure the same as the directories in the project, including root. Now the config builds only src and public. Src to the dist folder and public to root." />
+
+`mount` is part of the [Snowpack Configuration API](https://www.snowpack.dev/#configuration). It allows you to customize the file structure of your project. The key is the name of the directory and the value is where you'd like them in the final build. With this new configuration, Snowpack builds files in `public` like `public/index.css` directory into `index.css`. It builds files in `src` like `src/index.js` into `/_dist_/index.js`, so you'll need to change that path in your `index.html`:
+
+```diff
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack for configuration file changes. When you start up again, if it worked, it should look the same.
+
+## Styling your project
+
+Your site is a little plain, so in this step you'll add an image and animate it with CSS.
+
```

Comment:
maybe not "is a little plain" but instead "is looking a little plain"

Also nit: turn into two sentences:

```
Your site is looking a little plain. In this step you'll spruce it up with images and a CSS animation.
```